test/round-tracker.test.js

```diff
@@ -211,6 +215,15 @@ describe('Round Tracker', () => {
       assert.strictEqual(typeof startEpoch, 'number')
     })
   })
+
+  describe('startRoundTracker', () => {
+    it('detects the current round', async function () {
+      this.timeout(60_000)
```

Comment:
The pull request passed the CI but the deployment run timed out. I have increased the timeout in https://github.com/filecoin-station/spark-api/commit/4ba3f532d8b8e05f991d3e3bb718e12b056002f1